src/index.js

```
@@ -256,7 +262,6 @@ async function replaceInner(path, el, selector) {
     const resp = await fetch(path);
     if (!resp.ok) {
       // eslint-disable-next-line no-console
-      console.log('error loading content:', resp);
```

I'd keep this for debugging purpose so we know if the URL 404ed or not, otherwise it's going to be silently swallowed